src/client/components/friendsView/FriendMenuItemPage.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { useAppDispatch, useAppSelector } from '../../state/hooks';
+import allActions from '../../state/actions/allActions';
+import styled from 'styled-components';
+import {OrangeButton, OrangeNavbar, HeaderImage } from '../../styles/shared';
+import MenuItemIncrementor from '../Menu/MenuItemIncrementor';
+import { useHistory } from 'react-router-dom';
+
+const MainConatiner = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
+
+const CheckoutButton = styled(OrangeButton)`
+  border-radius: 7px;
+  width: 50%;
+`;
+
+export default function FriendMenuItemPage () {
+  const currentOrder = useAppSelector((state)=>state.currentMenuItem)
+  const item = useAppSelector((state)=>state.currentItemQuantityPrice)
+  const dispatch = useAppDispatch();
+  const totalPrice = item.price*item.count;
+  const history = useHistory();
+
+
+  function clickHandler () {
+    if(item.count===0){
+      return
+    } else {
+      dispatch(allActions.addToPriceTotal(totalPrice));//
+      dispatch(allActions.addItemToOrders(currentOrder));
+      history.push("/Menu/Friends");
+    }
+  }
+
+  React.useEffect(()=>{
+    dispatch(allActions.UpdateTotalPrice(totalPrice.toFixed(2)));
+  },[item])
+
+
+  return(
+    <MainConatiner>
+      <OrangeNavbar/>
+      <HeaderImage src="/Dannys_bg.png"/>
+      {/* <img src='Dannys_bg.png'/> */}
+      <h2>{currentOrder.itemName}</h2>
```

Comment:
this might throw an error "itemName" might need to change to "food"

Reply:
good catch, fixed